src/DemoFluentUI/Components/DarkModeFluentSwitch.razor

```diff
@@ -0,0 +1,37 @@
+﻿<FluentDesignTheme @bind-Mode="@Mode"
```

Comment:
I agree with you to upgrade to .NET9 but not to change the code about the Theme Switcher. The goal is to explain like in the demo made by Vincent during the Show, not to have all features. For that, we have the doc website and the doc source code. Keep simple os better 😀

Reply:
The demo made by Vincent is flawed, if you look at the console log you will see the following error, which may put people off using fluentui - 'Cannot read properties of null (reading 'mode')'. 

If you want reject this pull request and I will make another one with just the .Net 9 upgrade changes, but I would prefer if we had good theming without any errors.

![failure](https://github.com/user-attachments/assets/5a5fce54-1ec7-4378-a823-d77b42cc7eb0)

Reply:
Yes, it is flawed but it is ok for this usecase